ruby_on_rails/forms_and_authentication/form_basics.md

```diff
@@ -19,171 +19,183 @@ This section contains a general overview of topics that you will learn in this l
 
 ### Forms in HTML
 
-Step one is to be able to create a form in HTML.  Remember how that looks?
+Step one is to be able to create a form in HTML. Remember how that looks?
 
-~~~html
-  <form action="/somepath" method="post">
-    <input type="text">
-    <!-- other inputs here -->
-    <input type="submit" value="Submit This Form">
-  </form>
-~~~
+```html
+<form action="/somepath" method="post">
+  <input type="text" />
+  <!-- other inputs here -->
+  <input type="submit" value="Submit This Form">
+</form>
+```
 
-There are plenty of `input` tags to choose from, including `button`, `checkbox`, `date`, `hidden`, `password`, `radio` and many more (see [the full list from W3 Schools](https://www.w3schools.com/html/html_form_input_types.asp)).
+There are plenty of `input` tags to choose from, including `button`, `checkbox`, `date`, `hidden`, `password`, `radio` and many more (see [the full list of HTML input types from W3Schools](https://www.w3schools.com/html/html_form_input_types.asp)).
 
 ### Viewing what your form submits
 
-If you want to see what your forms are submitting to your Rails app, look through the output that gets printed into your console when you run your `$ rails server`.  Whenever you submit a very basic form for a user email signup, it should include lines that look something like:
+If you want to see what your forms are submitting to your Rails app, look through the output that gets printed into your console when you run your `$ rails server`. Whenever you submit a very basic form for a user email signup, it should include lines that look something like:
 
-~~~bash
+```bash
   Started POST "/user" for 127.0.0.1 at 2013-11-21 19:10:47 -0800
   Processing by UsersController#create as TURBO_STREAM
   Parameters: {"utf8"=>"✓", "authenticity_token"=>"jJa87aK1OpXfjojryBk2Db6thv0K3bSZeYTuW8hF4Ns=", "email"=>"foo@bar.com", "commit"=>"Submit Form"}
-~~~
+```
 
-*Note: this is from a form that might be generated by a Rails helper method, as explained in a later section below*
+_Note: this is from a form that might be generated by a Rails helper method, as explained in a later section below_
```

Comment:
We would need `*` instead of `_` here, but the MD036 error triggers because the entire line is emphasised, which people often use for headings instead of, well, actual headings.

The appropriate fix for this line would be:
```suggestion
**Note:** This is from a form that might be generated by a Rails helper method, as explained in a later section below.
```

Embolden the `Note:` bit and leave the rest normal. That should resolve the remaining errors.

Reply:
Got it, thanks for the tip and explanation. The linter passed this time!